ffmpeg.h

```diff
@@ -25,6 +25,12 @@
 
 #endif
 
+/*
+ * AVPixelFormat is a newer release of avutil PixelFormat is the old name
+ */
+#ifndef AVPixelFormat
```

Comment:
I'm not sure if I'm missing something but I've having a hard time getting my head around why it works.

Assuming I've understood any of this correctly, I think I'd probably prefer that we only #define AVPixelFormat if we are building against the older version of AVUtil that needs us to do so.